cli/src/mainframe.tsx

```diff
@@ -18,15 +20,49 @@ import { SellControl } from './trade/sell.js'
 import { BuyControl } from './trade/buy.js'
 import { PortfolioControl } from './positions/portfolio.js'
 import { BurnControl } from './trade/burn.js'
+import { commandOptions, type OptionKey } from './options.js'
 
 const Router = () => {
   const { input } = useContext(UserInputContext)
   const { wallet } = useWallet()
+  const force = useOption('force')
+  const sync = useOption('sync')
+  const [confirmInput, setConfirmInput] = useState<string>('')
+  const [confirmed, setConfirmed] = useState<boolean>()
+
   const matched = matchCommand(input)
+
   if (matched?.wallet && !wallet.address) {
     return <WalletRequired />
   }
+
+  if (input === CommandKeys.Portfolio && !sync) {
+    // pass
+  } else if (commandOptions[input as CommandKeys]?.force && !force) {
```

Comment:
I think the components have their own confirmation step (which displays and explains what is about to happen). They are more informative and intuitive, rather than having a generalized confirmation prompt here (which lacks specific details)

Reply:
What confirmation step? Can you plz take an example?

Reply:
https://github.com/polymorpher/pano/blob/6e5ac18be8080f46698fb1dddb267ca786cc557c/cli/src/trade/trade.tsx#L345

https://github.com/polymorpher/pano/blob/6e5ac18be8080f46698fb1dddb267ca786cc557c/cli/src/trade/buy.tsx#L33

Reply:
Done